src/test/java/seedu/duke/commands/EditCommandTest.java

```diff
@@ -39,6 +41,14 @@ public void sellItemTest() {
         editParser.run();
         editParser = new EditParser("upc/123 p/double", inventory);
```

Comment:
Should there be an assertion here? Seems that it is missing in your test over here